smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestRequestGenerator.kt

```diff
@@ -46,10 +42,8 @@ open class HttpProtocolUnitTestRequestGenerator protected constructor(builder: B
             }
         }
 
-        writer.write("let urlPrefix = urlPrefixFromHost(host: \$S)", test.host)
-        writer.write("let hostOnly = hostOnlyFromHost(host: \$S)", test.host)
```

Comment:
These two lines were causing compile warnings for being unused.  They only seemed to be used in setting up the old middleware stack.